CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+![](https://graphics.thomsonreuters.com/style-assets/images/logos/reuters-graphics-logo/svg/graphics-logo-color-dark.svg)
+
+# Contributing Guide
+
+## Why this?
+
+Most Reuters Graphics repos don't include (or need) contributing guidelines. The ones that do represent critical infrastructure. They also are the ones we most want wider contributions from the team to improve and maintain.
+
+This doc provides for a few simple guidelines to make sure changes are well considered and represent the best ideas for how to move our tools forward while opening up the opportunity for others to ship their next great idea.
+
+## Who can contribute?
+
+Contributions are always welcome from members of the Reuters Graphics team.
+
+If anyone outside our team ever starts using these components, we will happily review any PRs or issues, **BUT** if they are designed solely to benefit a use case that isn't ours, they likely won't be merged.
+
+## How should Reuters Graphics staff contribute?
+
+Code contributions should be made through the normal [GitHub Flow](https://www.w3schools.com/git/git_github_flow.asp#:~:text=The%20GitHub%20flow%20is%20a,Make%20changes%20and%20add%20Commits). Basically, make a branch and submit a pull request (PR) for any new features or changes.
+
+Before you make a PR, we recommend you create an issue on this repo describing what is missing, broken or could be added or improved. (We'll close that issue when we merge your PR.)
```

Comment:
Might make sense to lead with this and next graf so we're following the chronology we want them to follow: First make issue, then branch, code, PR if that makes sense